frontends/qt/Makefile

```diff
@@ -1,3 +1,14 @@
+SECURITYFLAGS = " -O2 -D_FORTIFY_SOURCE=2 "
```

Comment:
Same here: better to move the flags to the innermost scope so it's clear how and where they apply.

Reply:
It's meant to apply to everything inside of the `frontends/qt/` directory generally.

Does this set of flags interfere in any specific build target in `frontends`?